syntax = "proto3";

package api.chat.event;

message MemberAdded {
    uint64 user_id = 1;
}
